app/typer/sanity/sanityDokumentasjon.ts

```diff
@@ -3,6 +3,7 @@ import { ISanityDokument } from './sanity';
 export enum EApiKeysDokumentasjon {
   DOKUMENTASJON = 'Dokumentasjon',
   OVERSKRIFT = 'dokumentasjonOverskrift',
+  VEILEDNING = 'veiledning',
```

Comment:
Endre navnet på denne til f.eks. `infoMellomlagring` eller noe som er mer beskrivende for innholdet

Reply:
Endret navnet fra veiledning til infomellomLagring [84edbb6](https://github.com/bekk/nav-familie-endringsmelding/pull/118/commits/84edbb6881907c70313a42bb0d88a49336fe1769)